apps/bc-wallet-api-server/src/database/migrations/meta/0009_snapshot.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
looks like this is the same migration, but you should not change it here, I believe this migration will be skipped by the database

Reply:
I reran the migration after getting from develop-4sure